modules/virt-creating-service-cli.adoc

```diff
@@ -55,9 +55,14 @@ spec:
   selector:
     special: key <1>
   type: NodePort <2>
+  ports: <3>
+    protocol: TCP
+    port: 80
+    targetPort: 9376
 ----
 <1> Specify the label that you added to the `spec.template.metadata.labels` stanza of the `VirtualMachine` manifest.
 <2> Specify `ClusterIP`, `NodePort`, or `LoadBalancer`.
+<3> Specifies a collection of network ports and protocols that you want to expose from the virtual machine.
```

Comment:
**QE REVIEW NEEDED** Please verify that these parameters are required and the definitions are correct.

Reply:
looks good, also there is an optional parameter
```
  ports:
    nodePort: 30000
```
if it is not set - its value will be allocated dynamically 
however not sure how important to say about it..